<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="bfc097f8-51ad-47d0-a7aa-c1ef9e737dd3" name="Samuel@SAMUEL-PC 2013-04-21 18:46:41" runUser="Samuel-PC\Samuel" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="1f294c37-af87-48bd-bb70-7fb5cebd3076">
    <Description>Esta es la configuración de pruebas predeterminada para una ejecución de pruebas local.</Description>
    <Deployment enabled="false" runDeploymentRoot="Samuel_SAMUEL-PC 2013-04-21 18_46_41">
      <DeploymentItem filename="D:\Documents\Visual Studio 2010\Projects\2013-1\DAO.House\DAO.House\bin\Debug\DAO.House.exe" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2013-04-21T18:46:41.1831431-05:00" queuing="2013-04-21T18:46:41.2281457-05:00" start="2013-04-21T18:46:41.2331460-05:00" finish="2013-04-21T18:46:41.5991669-05:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="GetAllFromProductoTest" storage="d:\documents\visual studio 2010\projects\2013-1\dao.house\testproject\bin\debug\testproject.dll" id="81be2098-a2bb-fd52-62f0-98fe35e61381">
      <Execution id="51220f31-3b54-4a78-a88e-31722ee7fd73" />
      <TestMethod codeBase="D:/Documents/Visual Studio 2010/Projects/2013-1/DAO.House/TestProject/bin/Debug/TestProject.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestProject.ProductoRepositoryTest, TestProject, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="GetAllFromProductoTest" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Resultados que no están en la lista" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="Todos los resultados cargados" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="81be2098-a2bb-fd52-62f0-98fe35e61381" executionId="51220f31-3b54-4a78-a88e-31722ee7fd73" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="51220f31-3b54-4a78-a88e-31722ee7fd73" testId="81be2098-a2bb-fd52-62f0-98fe35e61381" testName="GetAllFromProductoTest" computerName="SAMUEL-PC" duration="00:00:00.1775981" startTime="2013-04-21T18:46:41.2361462-05:00" endTime="2013-04-21T18:46:41.5841661-05:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="51220f31-3b54-4a78-a88e-31722ee7fd73">
      <Output>
        <ErrorInfo>
          <Message>El método de prueba TestProject.ProductoRepositoryTest.GetAllFromProductoTest produjo la excepción: 
System.ArgumentOutOfRangeException: El índice estaba fuera del intervalo. Debe ser un valor no negativo e inferior al tamaño de la colección.
Nombre del parámetro: index</Message>
          <StackTrace>    en System.ThrowHelper.ThrowArgumentOutOfRangeException()
   en System.Collections.Generic.List`1.get_Item(Int32 index)
   en TestProject.ProductoRepositoryTest.GetAllFromProductoTest() en D:\Documents\Visual Studio 2010\Projects\2013-1\DAO.House\TestProject\ProductoRepositoryTest.cs:línea 78
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>